core/shared/platform/nuttx/nuttx_platform.c

```diff
@@ -19,6 +19,11 @@
 #define IRAM0_CACHE_ADDRESS_HIGH 0x44000000
 #define IRAM_ATTR locate_data(".iram1")
 
+#define INTERNAL_SRAM_1_DBUS_ADDRESS_LOW 0x3fc88000
+#define INTERNAL_SRAM_1_DBUS_ADDRESS_HIGH 0x3fcf0000
```

Comment:
INTERNAL_SRAM_1_DBUS_ADDRESS_HIGH is not used?

Reply:
right. it's just for completeness.